release_1.7/mtl_session02_see.md

```diff
@@ -102,5 +102,5 @@ output:
 <!-- Do/Done Tables -->
 | [<img src = "https://github.com/lzim/teampsd/blob/master/resources/icons/done.png" height = "80" width = "80">](#.) **Done** | [<img src = "https://github.com/lzim/teampsd/blob/master/resources/icons/do.png" height = "90" width = "90">](#.) **Do** |
 | --- | --- | 
-| [<img src = "https://raw.githubusercontent.com/lzim/teampsd/master/resources/logos/mtl_how_data_sm.png" height = "75" width = "110">](http://mtl.how/data_test) We selected the clinics that make up our team for the Team Data. [![](https://github.com/lzim/teampsd/blob/master/resources/gifs/mtl_1.7/session2_data_ui_2.gif)](#.)| [<img src = "https://raw.githubusercontent.com/lzim/teampsd/master/resources/logos/mtl_how_data_sm.png" height = "75" width = "110">](http://mtl.how/data_test) Review the HF, Diag, Enc and SP tabs in Team Data to find a patient (zoom in) and find a team trend (zoom out). [![](https://github.com/lzim/teampsd/blob/master/resources/gifs/mtl_1.7/session2_data_ui_4_viz_diag.gif)](#.)| 
+| [<img src = "https://raw.githubusercontent.com/lzim/teampsd/master/resources/logos/mtl_how_data_sm.png" height = "75" width = "110">](http://mtl.how/data_test) We selected the clinics that make up our team for the Team Data. [![](https://github.com/lzim/teampsd/blob/master/resources/gifs/mtl_1.7/session2_data_ui_2.gif)](#.)| [<img src = "https://raw.githubusercontent.com/lzim/teampsd/master/resources/logos/mtl_how_data_sm.png" height = "75" width = "110">](http://mtl.how/data_test) Review the HF, Diag, Enc and Team (SP) Flow tabs in Team Data to find a patient (zoom in) and find a team trend (zoom out). [![](https://github.com/lzim/teampsd/blob/master/resources/gifs/mtl_1.7/session2_data_ui_4_viz_diag.gif)](#.)| 
```

Comment:
@jamesmrollins I see that the FNR for Team (SP) Flow found it's way through the archived file for a 1.7 release. 

Could you ask DEV if there's a way to add something similar to a Git Ignore to the FNR action yml file to prevent it from searching certain folders/file paths?
It would keep the archived files untouched, save time in the pull request process for reviewers, and possibly improve the Action processing time.

FYI @mnallajerla

Reply:
I'm going to keep this one edit in for now (I believe if I edit it back to "SP", the pull request will lose track of this file since it will not contain any edits).

I'll document this request in #2510 Story I with this use case + screenshot.